source/transactions/tests/write-concern.yml

```diff
@@ -269,8 +268,7 @@ tests:
           documents:
             - _id: 1
             - _id: 2
-        result:
-          insertedIds: {0: 1, 1: 2}
+        # avoid assertion since result fields are inaccessible for unacknowledged write concern
```

Comment:
This command is part of a transaction so the driver ignores the w:0 write concern. A result *is* returned.